### PR TITLE
Picker components

### PR DESCRIPTION
**Pull request by quietbits**
- Updated SDS version to the latest
- Added `stellar-sdk`
- Generic `RadioPicker` component used to create more use-case-specific pickers:
  - `OrderPicker`
  - `IncludeFailedPicker`
  - `AssetPicker`
- `PubKeyPicker`

<details>
<summary>Demo video</summary>
<hr />

[lab-pickers-demo.webm](https://github.com/stellar/laboratory/assets/7346473/24deb24d-7e7b-4a55-986f-e47dafa5fce3)
</details>

### CODE REVIEW

File: src/components/FormElements/AssetPicker.tsx [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+        isExpanded={Boolean(
+          selectedOption &&
+            ["issued", "credit_alphanum4", "credit_alphanum12"].includes(
+              selectedOption,
+            ),
+        )}
+      >
+        <AssetPickerFields
+          id={id}
+          code={{
```
Comment (jeesunikim) [inline src/components/FormElements/AssetPicker.tsx:226]: note to myself to will follow this style!

File: src/components/FormElements/AssetPicker.tsx [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+      case "credit_alphanum12":
+        minLength = 5;
+        maxLength = 12;
+        break;
+      default:
+        minLength = 1;
+        maxLength = 12;
+    }
+
+    if (!code.match(/^[a-zA-Z0-9]+$/g)) {
```
Comment (jeesunikim) [inline src/components/FormElements/AssetPicker.tsx:189]: do you think this regex will be re-used?

File: src/components/FormElements/OrderPicker.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  onChange: (optionId: string | undefined, optionValue?: string) => void;
+  labelSuffix?: string | React.ReactNode;
+};
+
+export const OrderPicker = ({
+  id,
+  selectedOption,
+  onChange,
+  labelSuffix,
+}: OrderPickerProps) => {
```
Comment (jeesunikim) [inline src/components/FormElements/OrderPicker.tsx:15]: we can just do `( )` and remove `{ }` and `return`

**Review by jeesunikim — APPROVED** [review]
jus smol comment

File: package.json [hunk 4]
```diff
@@ -15,7 +15,7 @@
     "git-info": "rm -rf src/generated/ && mkdir src/generated/ && echo export default \"{\\\"commitHash\\\": \\\"$(git rev-parse --short HEAD)\\\", \\\"version\\\": \\\"$(git describe --tags --always)\\\"};\" > src/generated/gitInfo.ts"
   },
   "dependencies": {
-    "@stellar/design-system": "^2.0.0-beta.5",
+    "@stellar/design-system": "^2.0.0-beta.6",
```
Comment (jeesunikim) [inline package.json:18]: should this be `beta.7`

Comment (quietbits) [reply package.json:18]: No, this PR doesn't need `v7`. The parent branch will have it updated.

File: src/components/FormElements/AssetPicker.tsx (hunk 2 above)
Comment (quietbits) [reply src/components/FormElements/AssetPicker.tsx:189]: Can't think of any other use cases right now. :thinking: If it is used anywhere else, we'll make it a const. :+1:

File: src/components/FormElements/AssetPicker.tsx (hunk 1 above)
Comment (quietbits) [reply src/components/FormElements/AssetPicker.tsx:226]: We might need to remove `onBlur` piece and always validate on `onChange`. :sweat_smile: The current PR I'm working on will slightly change the form element API.

File: src/components/FormElements/OrderPicker.tsx (hunk 3 above)
Comment (quietbits) [reply src/components/FormElements/OrderPicker.tsx:15]: Good call! I'll update.

**Pull request merged**